Makefile

```diff
@@ -5,7 +5,7 @@ PROJECTS=libbeat $(BEATS)
 PROJECTS_ENV=libbeat filebeat metricbeat
 SNAPSHOT?=yes
 PYTHON_ENV?=$(BUILD_DIR)/python-env
-VIRTUALENV_PARAMS?=
+VIRTUALENV_PARAMS?=-p /usr/var/python2
```

Comment:
We should not change the defaults here as it will potentially break other things. Perhaps you can open a separate issue / PR to discuss this in more detail?